lib/RuntimePlugin.js

```diff
@@ -185,7 +186,10 @@ class RuntimePlugin {
 							? entryOptions.publicPath
 							: globalPublicPath;
 
-					if (publicPath === "auto") {
+					if (publicPath === "__webpack_dynamic_public_path__") {
```

Comment:
I don't like this part but didn't know how to attach additional information to the chunk to know that it requires a different type of public path module :/ Any ideas?